R/inherit-param-estimates.R

```diff
@@ -0,0 +1,142 @@
+#' Inherit parameter estimates
+#'
+#' Set the initial parameter estimates of a model using the estimates of a
+#' previously executed model.
+#'
+#' @param .mod new model object to overwrite.
+#' @param .parent_mod Either a model object, or path to a model to inherit properties from.
+#' @param .inherit type of estimates to inherit from parent model.
+#' @param .bounds Whether to keep or discard the existing bounds when setting
+#'   the initial estimates in THETA records.
+#' @param .representation Whether to keep alternative representation options like
```

Comment:
Just a drive-by comment: This function is getting the values itself with `get_{theta,omega,sigma}` (i.e. it's the "final estimates from a previous NONMEM run" case from the parameter description), so it should always be "reset".  (`nmrec::set_{theta,omega,sigma}`, on the other hand, do not know the source of the values, so it seemed to me like it had to be up to the caller to decide the behavior.)